nntrainer/tensor/cl_operations/blas_kernel_interface.cpp

```diff
@@ -170,20 +165,19 @@ void dotCl(Tensor const &input, Tensor const &m, Tensor &result,
     }
     /// case2: (M * K) X (K * 1)
     else if (N == 1) {
-      transA ? sgemv_cl(data, mdata, rdata, dim2, dim1, lda, context)
-             : sgemv_cl(data, mdata, rdata, dim1, dim2, lda, context);
+      trans ? sgemv_cl(data, mdata, rdata, dim2, dim1, lda, context)
```

Comment:
It seems that transA has been changed to trans and transB has been changed to trans_m.
Is there a reason why you didn't use trans_n or another name?

And in other functions, we use transA and transB.

Reply:
`trans_m` and `trans` are variables from the function parameter. I believe using them is more coherent.